suse_migration_services/units/prepare.py

```diff
@@ -49,6 +49,9 @@ def main():
     hosts_setup = os.sep.join(
         [root_path, 'etc', 'hosts']
     )
+    trust_anchors = os.sep.join(
+        [root_path, 'usr', 'share', 'pki', 'trust', 'anchors']
```

Comment:
The '/usr/.../anchors' could be the returning of a Default method but it's ok as it is.

Reply:
Well, update-ca-certificates does not have any location option, i.e. it will only update the trust root if the certificate to be added is in "/usr/share/pki/trust/anchors/". Thus having this hard coded here is IMHO preferable of a "defaults" implementation as the latter would imply that it can be customized.